internal/jobcontainers/jobcontainer.go

```diff
@@ -0,0 +1,408 @@
+package jobcontainers
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"regexp"
+	"sync"
+	"syscall"
+	"time"
+
+	"github.com/Microsoft/hcsshim/internal/queue"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+
+	"github.com/Microsoft/hcsshim/internal/jobobject"
```

Comment:
Is there a reasoning behind how the imports are broken into groups?

Reply:
None at all, must've happened after deleting an import